docs/logs/customizing-the-sdk/README.md

```diff
@@ -14,16 +14,25 @@ TODO
 
 ### IncludeScopes
 
-A "[scope](https://docs.microsoft.com/dotnet/core/extensions/logging#log-scopes)"
-is an `ILogger` concept that can group a set of logical operations and attach
-data to each log created as part of a set.
+Log
+[scope](https://docs.microsoft.com/dotnet/core/extensions/logging#log-scopes) is
+an `ILogger` concept that can group a set of logical operations and attach data
+to each log created as part of a set.
 
 `IncludeScopes` is off by default. Setting this to `true` will include all
 scopes with the exported `LogRecord`. Consult the individual `Exporter`
 docs to learn more about how scopes will be processed.
 
 See [Program.cs](Program.cs) for an example.
 
+> [!NOTE]
+> When using [`ILogger.BeginScope<TState>(TState
+state)`](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger.beginscope),
+it is highly recommended to pass in an instance of `List<KeyValuePair<string,
+object?>>` or `IReadOnlyList<KeyValue<string, object?>>` as the `state` for the
+best performance. When performance is not a critical requirement,
```

Comment:
nit: IReadOnlyList is interface so can't pass an instance of it.

Reply:
Good point, let me reword it.